CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ set(ALEVERSION "0.5")
 option(USE_SDL "Use SDL" OFF)
 option(USE_RLGLUE "Use RL-Glue" OFF)
 option(BUILD_EXAMPLES "Build Example Agents" ON)
+option(BUILD_CPP_LIB "Build C++ Shared Library" ON)
+option(BUILD_CLI "Build ALE Command Line Interface" ON)
```

Comment:
sorry—I accidentally switched flags part-way through the patch. Fixed in latest commit by `BUILD_ALE_BIN` -> `BUILD_CLI`